presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/MetastoreOperationStats.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive.metastore;
+
+import java.util.List;
+
+public class MetastoreOperationStats
+{
+    private final MetastoreOperationType operationType;
+    private final List<Integer> lastDataCommitTimes;
+
+    public MetastoreOperationStats(MetastoreOperationType operationType, List<Integer> lastDataCommitTimes)
+    {
+        this.operationType = operationType;
+        this.lastDataCommitTimes = lastDataCommitTimes;
+    }
+
+    public List<Integer> getLastDataCommitTimes()
+    {
+        return lastDataCommitTimes;
+    }
+
+    public MetastoreOperationType getOperationType()
+    {
+        return operationType;
+    }
```

Comment:
Is this used anywhere?

Reply:
No, Since this member is defined in the class,  I suppose we should provide a way to access it.